tests/vxlan/test_vnet_vxlan.py

```diff
@@ -0,0 +1,167 @@
+import json
+import logging
+import pytest
+import vnet_config as vc
+import yaml
+
+from datetime import datetime
+from jinja2 import Template
+from netaddr import IPAddress
+from os import path
+from time import sleep
+from tests.ptf_runner import ptf_runner
+from vnet_utils import *
+
+from tests.common.fixtures.ptfhost_utils import remove_ip_addresses, change_mac_addresses, \
+                                                copy_arp_responder_py, copy_ptftests_directory
+
+logger = logging.getLogger(__name__)
+
+@pytest.fixture(scope="module")
+def num_vnet(request):
+    return request.config.option.num_vnet
+
+@pytest.fixture(scope="module")
+def num_routes(request):
+    return request.config.option.num_routes
+
+@pytest.fixture(scope="module")
+def num_endpoints(request):
+    return request.config.option.num_endpoints
+
+@pytest.fixture(scope="module")
+def ipv6_vxlan_test(request):
+    return request.config.option.ipv6_vxlan_test
```

Comment:
Can this be combined into a `vnet_test_parameters` object/namedtuple/dict or something similar?